NO_GLAC_001

anders-kolstad opened the pull request:
Documentation for indicator NO_GLAC_001 Glacial area

anders-kolstad commented:
Please wait a little before reviewing this. I'm just making some changes.

anders-kolstad commented:
Ok, then it's ready again.

audhalbritter commented on indicators/NO_GLAC_001/R/NO_GLAC_001.qmd:
```diff
@@ -0,0 +1,1731 @@
+regions and geographical regions. We have defined a threshold value
+(X~60~) as 10% remaining glacier area inside each EA, based on a general
+assumption (from Island Biogeography Theory) that further loss of area
+will result in \>10% local species extinctions.
```
Maybe add a reference for this?

DrMattG replied:
I have fixed this now

audhalbritter commented on indicators/NO_GLAC_001/R/NO_GLAC_001.qmd:
```diff
@@ -0,0 +1,1731 @@
+## 5. Uncertainties
+
+There is some uncertainty in the mapping method,and this is said to be
+3%. There is also some uncertainty, or at least some arbitrariness, in
```
Does this 3% need a reference?

anders-kolstad replied:
Adding it now

audhalbritter commented on indicators/NO_GLAC_001/R/NO_GLAC_001.qmd:
```diff
@@ -0,0 +1,1731 @@
+
+The dataset is manually downloaded.
+
+URL: https://nve.brage.unit.no/nve-xmlui/handle/11250/2831053
```
This url is not working in the html. Same for dataset B

DrMattG replied:
I have fixed this and for dataset B

anders-kolstad replied:
I will making the link active

audhalbritter commented on indicators/NO_GLAC_001/R/NO_GLAC_001.qmd:
````diff
@@ -0,0 +1,1731 @@
+```
+
+```{r fig-sonerOgSeksjoner}
+#| fig-cap: "Distribution of area within each combination of bioclimatic sone and section (i.e. each bioclimatic region)."
````
Add description of what SO1-5 and SE1-5 mean. Like which once are dry/wet etc. Or replace the facet labels with a more intuitive labels. Oceanic, strong oceanic etc.?

anders-kolstad replied:
Done

audhalbritter commented on indicators/NO_GLAC_001/R/NO_GLAC_001.qmd:
````diff
@@ -0,0 +1,1731 @@
+
+```{r}
+#| eval: false
+#| code-summary: 'Create EA dataset'
````
I like that you name these code chunks. Makes it really easy to follow the workflow.

audhalbritter commented on indicators/NO_GLAC_001/R/NO_GLAC_001.qmd:
```diff
@@ -0,0 +1,1731 @@
+#  labs(x = "Bioclimatic region",
+#       y = "Total mountain area (m2)") +
+#  coord_flip()+
+#  theme_bw()
```
This code is commented out. Is it still needed?

anders-kolstad replied:
No, it's just for future reference. I added an annotation.

audhalbritter commented on indicators/NO_GLAC_001/R/NO_GLAC_001.qmd:
```diff
@@ -0,0 +1,1731 @@
+
+Of the sones, its mainly in the alpine sones (sone 5) where we find glaciers. 
+@fig-gradientIndicator show how the warmer sones also have a little lower indicator values,
+as expected.
```
Why do the glaciers in sone 3 not have any extent? Are they all lost? And why is their indicator value not much lower then or 0? Or am I missing something here?

anders-kolstad replied:
In @fig-EAglacArea we see there are no glaciers in zone 3, but that was just because the values were so small they didnt show. I turned to a log10 axis to avoid that kind of confusion. 

The indicator values for zone 3 are generally quite low, but they are not zero unless they are all lost (one case of that happening)

audhalbritter reviewed (commented):
This indicator looks good. The .qmd renders and the html looks good.
I have only some minor points in separate comments.

DrMattG commented on indicators/NO_GLAC_001/R/references.bib:
```diff
@@ -0,0 +1,90 @@
+	date = {1984},
+	journal = {Concepts and techniques in modern geography},
+	note = {Publisher: GeoBooks}
+}
```
Add 
@book{macarthur2001theory,
  title={The theory of island biogeography},
  author={MacArthur, Robert H and Wilson, Edward O},
  volume={1},
  year={2001},
  publi

anders-kolstad replied:
Since I changed the reasoning for the x60 value I actually dont need this ref in the end.

DrMattG commented on indicators/NO_GLAC_001/R/NO_GLAC_001.qmd:
```diff
@@ -0,0 +1,1731 @@
+ecosystem assets (EA) are defined as intersections of bioclimatic
+regions and geographical regions. We have defined a threshold value
+(X~60~) as 10% remaining glacier area inside each EA, based on a general
+assumption (from Island Biogeography Theory) that further loss of area
```
...(from Island Biogeography Theory;
@macarthur2001theory)

DrMattG commented on indicators/NO_GLAC_001/R/NO_GLAC_001.qmd:
```diff
@@ -0,0 +1,1731 @@
+
+The dataset is manually downloaded.
+
+URL: https://nve.brage.unit.no/nve-xmlui/handle/11250/2831053
```
URL: <https://nve.brage.unit.no/nve-xmlui/handle/11250/2831053>

anders-kolstad commented:
Thank @DrMattG  and @audhalbritter . I'll update the PR accounting for your suggestions. First, I updated the PR with some edits that I had done a while back, but forgot to merge the dev branch. This update was

- change X60 reference level from 10 to 40%, and at the same time remove the IBG reference/justification
- new terminology (i used the term ecosystem assets wring - changes it to 'strata')
- added a new pie chart (mainly for fun)

anders-kolstad commented:
Revision complete @DrMattG @audhalbritter

anders-kolstad commented:
> Revision complete @DrMattG @audhalbritter

@DrMattG or @audhalbritter , can you please look at this and merge if OK.

audhalbritter commented:
Hei @anders-kolstad this looks good. All changes seem to have been implemented and the html renders.

The only problem I see is that I get some warnings (section 8.1, Intersect EAs and glacier maps) when I render the html: Warning: attribute variables are assumed to be spatially constant throughout all geometries

The warnings are not in the html that is already in the repo, so maybe it's just because I ran it for the first time?
The code says that this "Runs only if the files are not cached, in which case they are simply read back."
Anyway it's nothing big and can probably be solved in the next round.

Should I merge the PR and make a stable html? Or do you want to change something first?
This is Version: 000.002. Is this correct or is it now Version: 000.003?
